README.md

```diff
@@ -1,2 +1,75 @@
-# turtle
-The fastest agile project management app
+# Turtle
+
+Turtle is an issue tracker for software developers.
+
+- easy to use (tasks are controlled by dragging and dropping them around)
```

Comment:
Now there is no syntactic connection between the previous sentence and the elements of the list.

Maybe add to the previous paragraph another sentence: "It is:". And then the list?